### PR TITLE
Add configurable blacklist time for many checks util

### PR DESCRIPTION
**Pull request by xism4**

### CODE REVIEW

**Comment by xism4** [comment]
> Remove this, it is already done by BungeeCord and will be added together with upstream

You mean http handler?

**Comment by xism4** [comment]
K, already did

File: proxy/src/main/java/ru/leymooo/botfilter/config/Settings.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public class Settings extends Config
     @Comment("Проверять ли на бота при заходе на сервер во время бот атаки, не зависимо проходил ли проверку или нет")
     public boolean FORCE_CHECK_ON_ATTACK = true;
     @Comment("Показывать ли онлайн с фильтра")
-    public boolean SHOW_ONLINE = true;
+    public boolean SHOW_ONLINE = false;
```
Comment (NoJokeFNA) [inline proxy/src/main/java/ru/leymooo/botfilter/config/Settings.java]:
```suggestion
    public boolean SHOW_ONLINE = true;
```
Please do this in another Pull-Request as it is something complete different.

File: proxy/src/main/java/ru/leymooo/botfilter/config/Settings.java [hunk 2]
```diff
@@ -209,6 +209,8 @@ public void reload(File file)
 
         @Comment("Отключить ли проверку для клиентов с Geyser-standalone? Тип авторищации должен быть floodgate.")
         public boolean SKIP_GEYSER = false;
+        @Comment("Как долго вы хотите, чтобы черный список был для игрока")
```
Comment (NoJokeFNA) [inline proxy/src/main/java/ru/leymooo/botfilter/config/Settings.java]: You should probably say that the time is in minutes

**Review by NoJokeFNA — CHANGES_REQUESTED** [review]

File: proxy/src/main/java/ru/leymooo/botfilter/config/Settings.java (hunk 2 above)
Comment (xism4) [reply proxy/src/main/java/ru/leymooo/botfilter/config/Settings.java]: > You should probably say that the time is in minutes

Well, my russian is translator basicly but k

**Review by NoJokeFNA — APPROVED** [review]
Looks good to me in general

**Pull request closed without merging**